website/docs/discover.md

```diff
@@ -0,0 +1,318 @@
+# Discover Podman Desktop in 10 minutes
+
+Podman Desktop is the central place to work with containers as a developer.
+
+The main purpose of Podman Desktop is to use the Docker, Podman and Kubernetes APIs
```

Comment:
```suggestion
The main purpose of Podman Desktop is to use Podman, Docker and Kubernetes endpoints
```

endpoints maybe instead of API? Unsure.

But would be good having 'podman' as first rather than docker.